apps/website/src/components/show-and-tell/ShowAndTellEntry.tsx

```diff
@@ -176,7 +178,40 @@ const Header = ({ entry, isPresentationView }: ShowAndTellEntryProps) => {
               />
             </Link>
           </>
-        ) : null}
+        )}
+        {entry.azaZoo && (
+          <>
+            {` — `}
+            <Link
+              href="https://www.aza.org/about-us"
+              className={classes(
+                "inline-flex items-center gap-1 text-[#e0684b]",
+                isPresentationView
+                  ? "group text-nowrap rounded-full bg-alveus-tan-900/95 p-1 text-3xl shadow-lg transition-all hover:scale-102 hover:bg-alveus-tan-900 focus:bg-alveus-tan-900"
+                  : "hover:underline focus:underline",
+              )}
+              target="_blank"
+              custom
+            >
+              <strong
+                className={classes(
+                  "bg-[#e0684b] bg-clip-text font-bold text-transparent",
```

Comment:
I'm doing it this way instead of using `text-[#e0684b]` because the font width looks different next to the volunteeringMinutes badge.

Reply:
the font color is simply the AZA logo color